windows-vm-O365/azuredeploy.json

```diff
@@ -1,229 +1,229 @@
+        ],
+        "tags": {
+          "displayName": "VMNic01"
+        },
+        "properties": {
+          "ipConfigurations": [
+            {
+              "name": "ipconfig1",
+              "properties": {
+                "privateIPAllocationMethod": "Dynamic",
+                "subnet": {
+                  "id": "[variables('vmSubnetRef')]"
+                },
+                "publicIPAddress": {
+                  "id": "[resourceId('Microsoft.Network/publicIPAddresses', variables('vmIP01Name'))]"
+                }
+              }
+            }
+          ]
+        }
+      },
+      {
+        "name": "[variables('vmName')]",
+        "type": "Microsoft.Compute/virtualMachines",
+        "location": "[variables('location')]",
+        "apiVersion": "2015-06-15",
+        "dependsOn": [
+          "[concat('Microsoft.Storage/storageAccounts/', variables('storageName'))]",
+          "[concat('Microsoft.Network/networkInterfaces/', variables('vmNicName'))]"
+        ],
+        "tags": {
+          "displayName": "VM01"
+        },
+        "properties": {
+          "hardwareProfile": {
+            "vmSize": "[variables('vmSize')]"
+          },
+          "osProfile": {
+            "computerName": "[variables('vmName')]",
+            "adminUsername": "[parameters('vmAdminUsername')]",
+            "adminPassword": "[parameters('vmAdminPassword')]"
+          },
+          "storageProfile": {
+            "imageReference": {
+              "publisher": "[variables('vmImagePublisher')]",
+              "offer": "[variables('vmImageOffer')]",
+              "sku": "[parameters('vmOSVersion')]",
+              "version": "latest"
+            },
+            "osDisk": {
+              "name": "VMOSDisk",
+              "vhd": {
+                "uri": "[concat('http://', variables('storageName'), '.blob.core.windows.net/', variables('vmStorageAccountContainerName'), '/', variables('vmOSDiskName'), '.vhd')]"
+              },
+              "caching": "ReadWrite",
+              "createOption": "FromImage"
+            }
+          },
+          "networkProfile": {
+            "networkInterfaces": [
+              {
+                "id": "[resourceId('Microsoft.Network/networkInterfaces', variables('vmNicName'))]"
+              }
+            ]
+          }
+        },
+        "resources": [
+          {
+            "name": "SetupOffice",
+            "type": "extensions",
+            "location": "[variables('location')]",
+            "apiVersion": "2015-06-15",
+            "dependsOn": [
+              "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'))]"
+            ],
+            "tags": {
+              "displayName": "SetupOffice"
+            },
+            "properties": {
+              "publisher": "Microsoft.Compute",
+              "type": "CustomScriptExtension",
+              "typeHandlerVersion": "1.4",

```

Comment:
It's there; you just need the publisher Microsoft.Azure.Extensions:
![image](https://cloud.githubusercontent.com/assets/1448593/18687055/2ff000b8-7f33-11e6-8559-c0820c9f69f2.png)